ui/tests/integration/components/masked-input-test.js

```diff
@@ -96,4 +103,56 @@ module('Integration | Component | masked input', function (hooks) {
     const unMaskedValue = document.querySelector('.masked-value').value;
     assert.strictEqual(unMaskedValue, this.value);
   });
+
+  test('it renders correctly with empty string provided as value', async function (assert) {
```

Comment:
This is something I'm personally trying to be better at 👉  being explicit when labeling tests.

`correctly` doesn't really tell us much about what this test is supposed to achieve. However saying something like `it renders a minus icon when an empty string is provided as a value` gives us a clearer indication of what the test is intended to check

Reply:
That makes sense! I'll change that before merging. Thanks!